src/Components/Cluster/Cluster.spec.ct.js

```diff
@@ -55,7 +77,10 @@ describe('cluster page', () => {
     cy.get(CLUSTER_HEADER).should('have.length', 1);
     // renders table component
     cy.get(RULES_TABLE).should('have.length', 1);
+    // test how many rows were rendered
+    cy.getAllRows().should('have.length', 10);
```

Comment:
You have the data, so it is better to count the number of expected rows from there directly. This way, the data can be expanded without affecting the test